packages/dpns-contract/schema/v1/dpns-contract-documents.json

```diff
@@ -1,5 +1,7 @@
 {
   "domain": {
+    "documentsMutable": false,
```

Comment:
Use a data trigger here, and have mutable: true, canBeDeleted: true
Also add transferable: 1, tradeMode: 1

Then use data triggers for now to restrict everything.